go.mod

```diff
@@ -66,10 +70,14 @@ require (
 )
 
 replace (
+	// FIXME gogo/googleapis must be v1.3.2 <BECAUSE>
 	github.com/gogo/googleapis => github.com/gogo/googleapis v1.3.2
+	// FIXME golang/protobuf must be v1.3.5 <BECAUSE>
 	github.com/golang/protobuf => github.com/golang/protobuf v1.3.5
 	// urfave/cli must be <= v1.22.1 due to a regression: https://github.com/urfave/cli/issues/1092
 	github.com/urfave/cli => github.com/urfave/cli v1.22.1
+	// FIXME genproto must be v0.0.0-20200224152610-e50cd9704f63 <BECAUSE>
 	google.golang.org/genproto => google.golang.org/genproto v0.0.0-20200224152610-e50cd9704f63
+	// FIXME olang.org/grpc be v1.27.1 <BECAUSE>
 	google.golang.org/grpc => google.golang.org/grpc v1.27.1
```

Comment:
These three replace rules (protobuf , genproto, grape) were added in https://github.com/containerd/containerd/commit/de059b11ba3aafe24d64a1245e1f8ceccdbccaa1 (https://github.com/containerd/containerd/pull/4760). Most likely, this was only done to keep the vendored versions the same as before switching to go.mod, but we should add an explanation (and check if this is still required)

Reply:
IIRC, bumping grpc will cause protobuf to be bumped, thus https://github.com/containerd/ttrpc/issues/62

Reply:
Ah! Yes, I recalled there were some specific issues with grpc, but forgot which ones that were (but the motivation for me to open this PR, because it's very easy to miss for users, and (IIRC), some of the incompatibilities may not "fail hard", but only occur "sometimes", so better to have clear warnings